### DISCUSSION TITLE
Docker 运行是否支持 设置 READ_BUFFER_SIZE 参数。

### DISCUSSION THREAD

**Discussion by MrQQQQQQQQQQ** [messense/aliyundrive-webdav#266] [Q&A] [answered]
目前使用 docker 运行，当观看部分4K蓝光片源（码率较高，文件大概50-60g）时，偶尔会缓冲。不知道是否可以通过 设置更大的 缓冲区来解决这个问题。

**Comment by messense** [comment, marked as answer]
参考 https://github.com/messense/aliyundrive-webdav/issues/85#issuecomment-945786375 自行增加 `--read-buffer-size` 参数。

**Comment by MrQQQQQQQQQQ** [comment]
![image](https://user-images.githubusercontent.com/8612380/150298160-cd829c85-b041-44c5-b62b-bdbe2c0a1c64.png)
使用的爱快软路由内的 docker 插件，设置启动参数后 启动不了。不知道是爱快 docker 的问题，还是我参数没设置对。

**Reply by messense** [reply]
请阅读文档而不是猜测：https://docs.docker.com/engine/reference/commandline/run/

**Comment by MrQQQQQQQQQQ** [comment]
爱快 docker插件的限制 ，不能使用 docker run..

**Reply by messense** [reply]
万变不离其宗，爱快再怎么限制，最终参数都是对应到 docker run 的参数，你只需要理解一下这些参数的对应关系，而不是非要用 docker run 来跑。

**Comment by MrQQQQQQQQQQ** [comment]
感谢大佬指导，我再研究研究。
